src/driver/src/DefaultDriver.cpp

```diff
@@ -188,58 +197,82 @@ std::string normalizeOutputFile(const std::string& str) {
 }  // namespace
 
 
-genny::driver::DefaultDriver::ProgramOptions::ProgramOptions(int argc, char** argv) {
+DefaultDriver::ProgramOptions::ProgramOptions(int argc, char** argv) {
     namespace po = boost::program_options;
 
-    po::options_description description{u8"🧞‍ Allowed Options 🧞‍"};
+    po::options_description progDescription{u8"🧞‍ Allowed Options 🧞‍"};
     po::positional_options_description positional;
 
     // clang-format off
-    description.add_options()
-        ("help,h",
-            "Show help message")
-        ("list-actors",
-            "List all actors available for use")
-        ("dry-run",
-            "Exit before the run step---"
-            "this may still make network connections during workload initialization")
-        ("metrics-format,m",
+    const auto runModeHelp = u8R"(
+Genny run modes
+    normal
+        Run the workload normally; default mode if no run mode is specified
+
+    dry-run
+        Exit before the run step -- this may still make network connections during workload initialization
+
+    evaluate
+        Print the evaluated YAML workload file with minimal validation
+
+    list-actors
+        List all actors available for use
+    )";
+
+    progDescription.add_options()
+            ("help,h",
+             "Show help message")
+            ("run-mode", po::value<std::string>()->default_value("normal"),
+             runModeHelp)
+            ("list-actors",
+                    "DEPRECATED. Please use the \"list-actors\" run mode.")
+            ("dry-run",
+                    "DEPRECATED. Please the \"dry-run\" run mode.")
+            ("metrics-format,m",
              po::value<std::string>()->default_value("csv"),
              "Metrics format to use")
-        ("metrics-output-file,o",
-            po::value<std::string>()->default_value("/dev/stdout"),
-            "Save metrics data to this file. Use `-` or `/dev/stdout` for stdout.")
-        ("workload-file,w",
-            po::value<std::string>(),
-            "Path to workload configuration yaml file. "
-            "Paths are relative to the program's cwd. "
-            "Can also specify as first positional argument.")
-        ("mongo-uri,u",
-            po::value<std::string>()->default_value("mongodb://localhost:27017"),
-            "Mongo URI to use for the default connection-pool.")
-    ;
+            ("metrics-output-file,o",
+             po::value<std::string>()->default_value("/dev/stdout"),
+             "Save metrics data to this file. Use `-` or `/dev/stdout` for stdout.")
+            ("workload-file,w",
+             po::value<std::string>(),
+             "Path to workload configuration yaml file. "
+             "Paths are relative to the program's cwd. "
+             "Can also specify as the last positional argument.")
```

Comment:
similarly this seems like it "wants" to be the first positional param for the `genny run` subcommand:

```sh
genny run ./workloads/foo.yml
```

(I'm saying all this without really knowing how fancy or not `boost::program_options` is...it gets messy even in Python so I wouldn't be surprised if this makes things go off the rails.)